Doctrine/Orm/Filter/OrderFilter.php

```diff
@@ -118,13 +120,13 @@ public function getDescription(ResourceInterface $resource)
         }
 
         foreach ($properties as $property => $order) {
-            if (!$this->isPropertyMapped($property, $resource)) {
+            if (!$this->isPropertyEnabled($property) || !$this->isPropertyMapped($property, $resource)) {
```

Comment:
`isPropertyEnabled` really is a redundant check here. It will always return `true`.